packages/grafana-schema/src/common/table.cue

```diff
@@ -67,12 +67,17 @@ TableColoredBackgroundCellOptions: {
 	mode?: TableCellBackgroundDisplayMode
 } @cuetsy(kind="interface")
 
+TableCustomCellOptions: {
+	type: TableCellDisplayMode & "custom"
+	cellComponent: string
```

Comment:
@malcolmholmes or @sdboyer, how should this be typed if this should be a function (react component). Cue does not allow that which sort of makes sense as it is not serializable (I assume that is the reason). This should be used only in code not as something to pass from the backend. Should this just be defined separately? I would say this won't ever be used from backend/Go or serialized anyway.

Reply:
Define it separately. The schema defines what travels through the API boundary